Example/Source/Examples/Simple/RestrictedBrickViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  RestrictedBrickViewController.swift
+//  BrickKit-Example
+//
+//  Created by Peter Cheung on 7/31/17.
+//  Copyright © 2017 Wayfair LLC. All rights reserved.
+//
+
+import Foundation
+import BrickKit
+
+class RestrictedBrickViewController: BrickViewController, HasTitle {
+    
+    class var brickTitle: String {
+        return "Restricted Example"
+    }
+    
+    class var subTitle: String {
+        return "Basic example of using bricks with a restricted maximum and minimum height"
+    }
+    
+    override func viewDidLoad() {
```

Comment:
Can we also add an example that changes from `.restricted` > `.auto` to simulate the expand/collapse behavior?